docs/source/io_formats/index.rst

```diff
@@ -1,13 +1,15 @@
-.. _usersguide_output:
+.. _io_file_formats:
 
-===================
-Output File Formats
-===================
+===============
+IO File Formats
+===============
 
 .. toctree::
    :numbered:
    :maxdepth: 3
 
+   data_wmp
+   mgxs_library
```

Comment:
It's a little weird to me to have an XML format here alongside a bunch of binary formats.